src/components/modern/TextWithCopy.js

```diff
@@ -40,26 +40,29 @@ type Props = {|
   textToCopy?: string,
   style?: ViewStyleProp,
   iconColor?: string,
+  ensCopy?: boolean,
```

Comment:
Pls provide `toastText: string` instead of boolean flag here and update call sites with proper translation reference.

Reply:
Done.